src/components/plots/styling/violin/ViolinControls.jsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable import/no-unresolved */
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Collapse, Radio, Form, Slider, Skeleton,
+} from 'antd';
+import { useSelector } from 'react-redux';
+import SelectData from 'components/plots/styling/violin/SelectData';
+import SingleGeneSelection from '../SingleGeneSelection';
+
+const { Panel } = Collapse;
+
+const ViolinControls = (props) => {
+  const { config, onUpdate, setSearchedGene } = props;
+  const cellSets = useSelector((state) => state.cellSets);
```

Comment:
Can you call this under `index` and pass in `cellSets` as a prop? That would reduce coupling between components,